_data/hosting_providers.json

```
@@ -1,4 +1,14 @@
 [
+  {
+    "name": "Pantheon",
+    "link": "https://pantheon.io/",
+    "category": "full",
+    "tutorial": "",
+    "announcement": "https://pantheon.io/blog/pantheon-launches-global-cdn-automated-https-all-sites",
+    "plan": "",
+    "reviewed": "",
```

Ok, looks great so let's mark today as the reviewed date and get this in!
```suggestion
    "reviewed": "2019.6.28",
```